installer/install.iss

```diff
@@ -1587,10 +1587,14 @@ begin
     end;
 
     AppDir:=ExpandConstant('{app}');
+    if Builtin then
```

Comment:
I've received a compile error here, adding a `begin` to the end of the line fixed it.